src/main/java/seedu/address/logic/commands/staff/ListStaffCommand.java

```diff
@@ -27,7 +27,7 @@
             + "If fields are not specified, then all staff will be listed.\n"
             + "Parameters: [" + PREFIX_NAME + "NAME_1 NAME_2...] "
             + "[" + PREFIX_APPOINTMENT + "APPOINTMENT_1 APPOINTMENT_2...]\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie a/cook\n";
+            + "Example: " + COMMAND_WORD + " n/alice bob charlie a/cook\n";
```

Comment:
should use PREFIX_NAME instead of hardcoding n/
The a/ also

Reply:
fixed